src/applications/travel-pay/components/TravelClaimDetails.jsx

```diff
@@ -0,0 +1,137 @@
+        } catch (error) {
+          setClaimsError(error);
+        }
+      };
+
+      if (!claimDetails) {
+        fetchClaim();
+      }
+    },
+    [claimDetails, id],
+  );
+
+  if (isLoading) {
+    return (
+      <div className="vads-l-grid-container vads-u-padding-y--3">
+        <va-loading-indicator
+          label="Loading"
+          message="Please wait while we load the application for you."
+          data-testid="travel-pay-loading-indicator"
+        />
+      </div>
+    );
+  }
+
+  const {
+    createdOn,
+    claimStatus,
+    claimNumber,
+    appointmentDateTime,
+    facilityName,
+    modifiedOn,
+  } = claimDetails || {};
+
+  const [appointmentDate, appointmentTime] = formatDateTime(
+    appointmentDateTime,
+  );
+  const [createDate, createTime] = formatDateTime(createdOn);
+  const [updateDate, updateTime] = formatDateTime(modifiedOn);
+
+  return (
+    <>
+      {/* Q: Is the Mhv nav needed here? */}
+      {/* <MhvSecondaryNav /> */}
+
+      <article className="usa-grid-full vads-u-padding-bottom--0">
+        {/* <div className="claim-details-breadcrumb-wrapper"> */}
+        {/* <va-icon className="back-arrow" icon="arrow_back" /> */}
+        <BreadCrumbs />
+        {claimDetails && (
+          <>
+            <h1>Your travel reimbursement claim for {appointmentDate}</h1>
+            <h2>Claim number: {claimNumber}</h2>
+            <h3>Where</h3>
+            <p>
+              {appointmentDate} at {appointmentTime} appointment
+            </p>
+            <p>{facilityName}</p>
+            <h3>Claim status: {claimStatus}</h3>
+            <p>
+              Submitted on {createDate} at {createTime}
+            </p>
+            <p>
+              Updated on on {updateDate} at {updateTime}
+            </p>
+          </>
+        )}
+
+        {claimsError && <h1>There was an error loading the claim details.</h1>}
+
+        <hr />
+
+        <p>
+          If you're eligible for reimbursement, we'll deposit your reimbursement
+          in your bank account.
+        </p>
+
+        <a
+          href="https://whereDoesThisGo?.com"
+          className="vads-u-display--inline-block vads-u-margin-bottom--4"
+        >
+          Learn how to set up direct deposit for travel pay reimbursement
+        </a>
+        <va-need-help>
+          <div slot="content">
+            <p>
+              To manage your travel claims, file a new claim, or learn what your
+              claim status means, go to our{' '}
+              <a className="btsss-portal-link" href={BTSSS_PORTAL_URL}>
+                Beneficiary Travel Self Service System (BTSSS) portal (opens in
+                new tab)
+              </a>
+              .<br />
+              Or call <va-telephone contact="8555747292" /> from 7 a.m. to 7
+              p.m. Monday through Friday. Have your claim number ready to share
+              when you call.
+            </p>
+          </div>
+        </va-need-help>
```

Comment:
We should pull this out into its own component. 

1. We've used it a couple times now
2. The portal (for Veterans) will eventually be replaced

Reply:
I was planning on doing that as part of clean-up, but also:

> We've used it a couple times now

This is not 100% true. There's a few discrepancies between this and the same section on the other page. This might just need to be combined, but something to validate with @micahtaylor et al. To my reading, it seems like it could be a single component with the same language